### ISSUE TITLE
Break after upgrading to 9.8.2 - Failed to execute 'observe' on 'MutationObserver'

### ISSUE THREAD

**Issue by rafaellehmkuhl** [vueuse/vueuse#2785]
### Describe the bug

After upgrading, I got the messages below. I was in version 9.1.1 and tracked the bug down to 9.8.2 (it is not present in 9.8.1).

<img width="618" alt="image" src="https://user-images.githubusercontent.com/6551040/219710825-60a28254-a315-4b5d-b4c1-1557fd4c2277.png">

The main error seems to be: `Failed to execute 'observe' on 'MutationObserver': parameter 1 is not of type 'Node'.`

I'm using data from `useElementBounding` while mounting this component, and use also those other composables in the same component:

`useConfirmDialog, useElementBounding, useElementHover, useMouseInElement, useWindowSize`

PS: I couldn't reproduce the bug in StackBlitz yet. I'm working on that, but opened the issue in hope someone can help just by the console error.

### Reproduction

https://stackblitz.com/edit/vitejs-vite-djhaji?file=src/App.vue

### System Info

```Shell
System:
    OS: macOS 12.6
    CPU: (8) arm64 Apple M1
    Memory: 211.00 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 19.5.0 - /opt/homebrew/bin/node
    Yarn: 1.23.0-20220130.1630 - /opt/homebrew/bin/yarn
    npm: 9.3.1 - /opt/homebrew/bin/npm
  Browsers:
    Chrome: 109.0.5414.119
    Firefox: 107.0.1
    Safari: 15.6.1
  npmPackages:
    @vueuse/core: ^9.12.0 => 9.12.0
    vue: ^3.2.33 => 3.2.33
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by rafaellehmkuhl** [comment]
Maybe a duplicate of #2782, as I also use `useStorage` vastly in the project.

**Comment by rafaellehmkuhl** [comment]
I can confirm that removing the usage of `useStorage` fixes my problems, so the problem is indeed in it and a duplicate of #2782.

**Comment by rafaellehmkuhl** [comment]
It appears the problem was fixed in version 9.13.0.

**Comment by rafaellehmkuhl** [comment]
> It appears the problem was fixed in version 9.13.0.

Correction: the problem was not fixed. It was just not happening to me because I already had something in the localStorage. By cleaning the localStorage (and so with `useStorage` trying to populate it) the problem remains.
